.github/workflows/license-year-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Update copyright year in license files
+
+on:
+  schedule:
+    - cron: "0 3 1 1 *"
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions:checkout@v2
+        with:
+          fetch-dept: 0
+      - uses: FantasticFiasco/action-update-license-year@v2
```

Comment:
Are you sure that it supports our license? From the [description](https://github.com/FantasticFiasco/action-update-license-year) it seems like it doesn't:

![image](https://user-images.githubusercontent.com/20105593/126497611-0a316743-db04-4811-bac8-4589f70b72c4.png)

Reply:
oh, I'am sorry, I forgot this. It works for `client`, but for server I should rewrite this for our license

Reply:
it works for our license, [proof](https://github.com/Avvessalom/test-project/pull/13/files)